provider/internal/image.go

```diff
@@ -701,20 +696,29 @@ func (i *Image) Create(
 
 	cli, err := i.client(ctx, state, input)
 	if err != nil {
-		return id, state, err
+		return infer.CreateResponse[ImageState]{ID: id, Output: state}, err
```

Comment:
In some places we call this State and in other places it's Output, which feels a little confusing.